test/defn/k-files/hash-mint.k

```diff
@@ -3,15 +3,20 @@ module HASH-MINT
 
   syntax MInt{160}
   syntax MInt{256}
+  syntax MInt{128}
 
   configuration <k> $PGM:K </k>
-                <state> 0p256 |-> 1p256 0p160 |-> 1p160 </state>
+                <state> 0p256 |-> 1p256 0p160 |-> 1p160 0p128 |-> 1p128 </state>
 
-  syntax KItem ::= foo() | bar()
+  syntax KItem ::= foo() | bar() | baz()
 
   rule <k> foo() => bar() ~> State [ 0p256 ] ...</k>
        <state> State </state>
+  rule <k> bar() => baz() ~> State [ 0p160 ] ...</k>
+       <state> State </state>
   rule <k> bar() => State [ 0p160 ] ...</k>
```

Comment:
you probably need to delete this line, or else the test will be non-deterministic.

Reply:
Even with the `[owise]` attribute?

Reply:
sorry I missed the `owise` attribute but it would make the rule redundant anyway. I see that you deleted so all good. You still need to regenerate the KORE for the test though.

Reply:
Good point, thanks!